.github/workflows/ci_pr_tests.yml

```diff
@@ -24,9 +24,10 @@ jobs:
         run: pg-start ${{ matrix.pg }}
       - name: Check out the repo - PG ${{ matrix.pg }}
         uses: actions/checkout@v4
-      - name: 🧪 Test on PG ${{ matrix.pg }}
+      - name: 🧪 Test - PG ${{ matrix.pg }}
         run: pgrx-build-test
       - name: 📎 Clippy - PG ${{ matrix.pg }}
+        if: ${{ matrix.pg == '17' }}
```

Comment:
Only run for the latest version, we should not have big differences on other versions, and it should be enough.